src/new-components/Faq.vue

```diff
@@ -0,0 +1,302 @@
+<template>
+  <div class="faq" id="faqScroll">
+    <h1 id="desktop-faq">FAQ - Frequently Asked Questions</h1>
+    <h1 id="mobile-faq">FAQ</h1>
+    <div class="faq-container">
+      <div class="categories">
+        <div
+          v-for="name in Object.keys(faqData)"
+          :key="name"
+          :class="{ 'category-bubble': true, selected: selected === name }"
+          
+        >
+          <div :style="{ width: '100%' }" @click="selectCategory(name)">
```

Comment:
Noticed a usability issue due to placing the click handler here: if you click just outside of the text (i.e. the green area in the attached image), I'd expect it to switch to that category, but it'll only switch if you click within the blue area.

![image](https://user-images.githubusercontent.com/18533760/95158084-bb948c80-0768-11eb-80b4-0f93dd4fb5de.png)